app/controllers/public/index.js

```diff
@@ -4,6 +4,12 @@ const { Controller, computed, String } = Ember;
 
 export default Controller.extend({
 
+  speakers: [{  name: 'Speaker 1', organisation: 'Org 1' },
+  { name: 'Speaker 2', organisation: 'Org 2', links: [{ link: 'facebook', url: '#' }] },
```

Comment:
Call it `socialLinks` instead of `links`. That how the API response is gonna be.